scripts/config_dealii.sh

```diff
@@ -56,6 +56,7 @@ cmake \
     -D DEAL_II_CXX_FLAGS_RELEASE="-O3" \
     -D DEAL_II_CXX_FLAGS_DEBUG="-Og" \
     -D DEAL_II_WITH_MPI:BOOL="ON" \
+    -D DEAL_II_WITH_HDF5:BOOL="ON" \
```

Comment:
ExaDG should compile without errors if this variable is set `OFF`. Is this guaranteed?

Reply:
When trying to use the new functionality of this MR in ExaDG, an error message when running the executable should tell the user that one has to activate HDF5 in the configuration files to use this postprocessing functionality.

Reply:
I will check this, but it will take some time to compile deal.ii without hdf5 to test it.